src/utils/clearSession.ts

```diff
@@ -0,0 +1,9 @@
+const clearSession = () => {
+  localStorage.removeItem('token');
+  localStorage.removeItem('currentRole');
+  localStorage.removeItem('user');
+  localStorage.removeItem('expToken');
+  localStorage.removeItem('impersonatingUserId');
+};
```

Comment:
Is there anything put into localStorage that we do want to persist between sessions? If there aren't, could this be replaced with `localStorage.clear()`?

Reply:
Although clear(), would also work, for now, I think it's safer to remove the specific items.

Reply:
I don't feel that manual removal is safer. If someone in the future copies how we set a session variable without being aware of the need to clear it that could lead to bugs. Using `clear()` eliminates that risk.

Reply:
Yes, I see your point and had exactly the same considerations. What I also considered is that localStorage could contain other data not related to sessions that some features might want to persist. For example right now if clear() was added it would alter the functionality of the drawer, whose state of open/close is persisted in localstorage even if you log out and back in again. We can have a small discussion in the meeting. Another option would be to implement some `SessionManager` that contains the logic

Reply:
Can we add a comment to `clearSession` explaining this so future devs can see more easily what should be listed in here and why we don't just use `localStorage.clear()`

Reply:
Done